lighthouse-core/report/templates/report-template.html

```diff
@@ -187,9 +183,7 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                   {{/if}}
 
                   <div class="subitem-result">
-                    {{#if subItem.comingSoon}}
-                          <span class="subitem-result__unknown score-unknown-bg">N/A</span>
-                    {{else if subItem.error}}
+                    {{#if subItem.error}}
```

Comment:
nuked that rule.


also here's the unused CSS against our report: 
![image](https://cloud.githubusercontent.com/assets/39191/22672524/65344298-ec89-11e6-8738-7ee857f87dc4.png)

haha